spaceship.zsh

```diff
@@ -642,6 +642,8 @@ spaceship_ruby() {
     ruby_version=$(chruby | sed -n -e 's/ \* //p')
   elif _exists rbenv; then
     ruby_version=$(rbenv version-name)
+  elif _exists ruby; then
```

Comment:
Does asdf modify global version of ruby? Well, in that case, this seems reasonable?

Reply:
Hi, yes it does!
asdf modifies ruby's version for every folder
<img width="450" alt="imagen" src="https://user-images.githubusercontent.com/3658902/32025904-37adadf6-b9e2-11e7-9743-f7d952fd446e.png">

Reply:
What I would change is the way that spaceship gets current ruby version and the condition.

Instead of `elif _exists ruby; then` I would use `elif _exists asdf; then`

And instead of ` ruby_version=$(ruby -v | awk '{print $2}')` you can obtain ruby version using `current` asdf option:

`asdf current ruby | awk '{print $0}'`

![image](https://user-images.githubusercontent.com/6479554/32045603-f4f93624-ba40-11e7-996f-f751a9cf4218.png)